src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Be careful, the excavator started digging a hole!!!");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("The excavator  stopped working, you can install the formwork.");
```

Comment:
Similarly, the message in the 'stopWork' method should be more informative and indicate that it's the 'Excavator' that has stopped working. You could rewrite it as 'Excavator has stopped working - safe to install the formwork.'